FILMING.md

```diff
@@ -0,0 +1,19 @@
+# Notice of filming
+
+Our Networks 2019 is an area where photography, audio, and video recording may occur.
+
+By entering, and by your attendance here, you acknowledge that you are aware of the presence of a live stream and the possibility of being asked to be photographed, filmed, and/or otherwise recorded. 
+
+## Live stream
+
+Main room sessions will be live streamed and video recordings will be uploaded after the event. Audio from audience microphones will be included in the live stream. Video recordings will be released under a non-commercial Creative Commons license. Organizers will identify areas outside of the live stream camera shot for those who do not wish to be included.
+
+## Taking photos or recording audio/video
+
+Do not photograph, video, or audio record anyone at Our Networks without their express permission, sought in advance. Before taking a photo, filming, and/or otherwise recording, organizers require that you ask everybody identifable in the picture if they agree to be photographed or filmed. 
```

Comment:
```suggestion
Do not photograph, video, or audio record anyone at Our Networks without their express permission, sought in advance. Before taking a photo, filming, and/or otherwise recording, organizers require that you ask everybody identifiable in the picture if they agree to be photographed or filmed. 
```